frontend/src/metabase-types/store/mocks/state.ts

```diff
@@ -19,25 +18,30 @@ import { createMockSettingsState } from "./settings";
 import { createMockSetupState } from "./setup";
 import { createMockUploadState } from "./upload";
 
-export const createMockState = (
-  opts?: Partial<State> | Partial<EnterpriseState> | Partial<SdkStoreState>,
-): State => ({
-  admin: createMockAdminState(),
-  app: createMockAppState(),
-  auth: createMockAuthState(),
-  currentUser: createMockUser(),
-  dashboard: createMockDashboardState(),
-  embed: createMockEmbedState(),
-  entities: createMockNormalizedEntitiesState(),
-  metabot: createMockMetabotState(),
-  parameters: createMockParametersState(),
-  qb: createMockQueryBuilderState(),
-  requests: createMockRequestsState(),
-  routing: createMockRoutingState(),
-  settings: createMockSettingsState(),
-  setup: createMockSetupState(),
-  upload: createMockUploadState(),
-  modal: null,
-  sdk: createMockSdkState(),
```

Comment:
not forcing `sdk` for non sdk tests and rely on passing `sdk` prop for SDK tests. It seem all SDK tests are passing `sdk` prop anyway.